app/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,8 @@
 
                 <!--Add in your scheme/host from registered redirect URI-->
                 <data android:scheme="msauth"
-                    android:host="<PACKAGE_NAME>/<SIGNATURE HASH> - must be registered at https://aka.ms/MobileAppReg" />
+                    android:host= "<PACKAGE_NAME> - must be registered at https://aka.ms/MobileAppReg"
```

Comment:
The package name must be in lower case.

Reply:
I think we'll try to apply this when configuring in the portal. Need to discuss on Monday w/ Portal team.